packages/lesswrong/components/form-components/FormComponentPostEditorTagging.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Components, registerComponent } from '../../lib/vulcan-lib';
+import toDictionary from '../../lib/utils/toDictionary';
+import mapValues from 'lodash/mapValues';
+import { taggingNamePluralCapitalSetting } from '../../lib/instanceSettings';
+
+/**
+ * Edit tags on the new or edit post form. If it's the new form, use
+ * TagMultiSelect; a server-side callback will convert tags to tag-relevances.
+ * If it's the edit form, instead use FooterTagList, which has the same
+ * voting-on-tag-relevance as the post page. Styling doesn't match between these
+ * two, which is moderately unfortunate.
+ */
+const FormComponentPostEditorTagging = ({value, path, document, label, placeholder, formType, updateCurrentValues}: {
```

Comment:
I'm still concerned that this is not a great mobile experience, but I assume it would be hard to hide this field on mobile and it's technically possible to downvote a tag when editing a post (so you can undo an accidental tag selection), so probably ok to leave it.

Reply:
Nod. I'd prefer to err on the side of leaving mobile able to do it at-all, but we can revisit if it turns out to be annoying.